region/maridia/inner-yellow/Plasma Spark Room.json

```diff
@@ -507,6 +507,70 @@
     },
     {
       "link": [2, 3],
+      "name": "Leave with Spark",
+      "requires": [
+        "Gravity",
+        "SpeedBooster",
+        {"enemyKill": {
+          "enemies": [["Owtch", "Owtch"]]
+        }},
+        "canShinechargeMovementComplex",
+        {"canShineCharge": {
+          "usedTiles": 16,
+          "openEnd": 2,
+          "gentleDownTiles": 2,
+          "gentleUpTiles": 2,
+          "steepDownTiles": 1,
+          "steepUpTiles": 1,
+          "startingDownTiles": 1
+        }},
+        {"shinespark": {"frames": 10}}
+      ],
+      "exitCondition": {
+        "leaveWithSpark": {}
+      },
+      "note": [
+        "Gain a shinecharge by running right-to-left on the underwater runway on the right side of the room.",
+        "Perform quick ledge-grabs to reach the platform in front of the door in time to spark out."
+      ],
+      "devNote": "It's technically possible to evade the Owtches without killing them, but it's unclear if this can be done reliably."
+    },
+    {
+      "link": [2, 4],
+      "name": "Shinespark",
+      "requires": [
+        "Gravity",
+        "SpeedBooster",
+        {"or": [
+          "HiJump",
+          "canSpringBallJumpMidAir"
+        ]},
+        {"enemyKill": {
+          "enemies": [["Owtch", "Owtch"]]
+        }},
+        "canShinechargeMovementComplex",
+        {"canShineCharge": {
+          "usedTiles": 16,
+          "openEnd": 2,
+          "gentleDownTiles": 2,
+          "gentleUpTiles": 2,
+          "steepDownTiles": 1,
+          "steepUpTiles": 1,
+          "startingDownTiles": 1
+        }},
+        {"shinespark": {
+          "frames": 43,
+          "excessFrames": 3
+        }}
+      ],
+      "note": [
+        "Gain a shinecharge by running right-to-left on the underwater runway on the right side of the room.",
+        "Use HiJump or a Spring Ball jump to quickly exit the water to the left and spark up."
+      ],
+      "devNote": "It's technically possible to evade the Owtches without killing them, but it's unclear if this can be done reliably."
```

Comment:
```suggestion
      "devNote": "FIXME: It's technically possible to evade the Owtches without killing them, but it's unclear if this can be done reliably."
```